test/models/product_content_test.rb

```diff
@@ -36,6 +36,10 @@ def test_repositories
       assert_equal repo.clones.count + 1, @product_content.repositories.size
 
       assert_includes @product_content.repositories, repo
+
+      @product_content.repositories.each do |clone|
+        assert clone.in_default_view?
+      end
```

Comment:
Thanks for adding an assertion to verify your fix. Based on our IRC discussion I advise removing the assertion above which checks the count of the repositories. I think it's meaningless and I regret writing that :-1: 

Your new assertion is much better and defines the behavior of the change you made. You might also add an assertion in the loop which verifies that the product on each repo matched the product_content object ie

```
assert_equal @product_content.product, clone.product
```

Maybe `repo` is a better name than `clone` since it describes the type of object being iterated over?

Reply:
Okay, I'll remove the previous assertion.  And yeah I like the name `repo` better, I just wasn't sure if that would cause confusion with the other repo that's also being tested.  I'll change :+1: